doc/_stdlib_gen/stdlib-content.jsonnet

```diff
@@ -109,6 +109,7 @@ local html = import 'html.libsonnet';
               <ul><code>std.asin(x)</code></ul>
               <ul><code>std.acos(x)</code></ul>
               <ul><code>std.atan(x)</code></ul>
+              <ul><code>std.atan2(y, x)</code></ul>
```

Comment:
Agree with the location, but IMHO we should include a note mentioning when it was added. It doesn't have to be put into the `availableSince` field, I think we can just include a parenthetical note in the text on this line. e.g.,

```
<ul><code>std.atan2(y, x)</code> (available since 0.21.0)</ul>
```

There are some other places (e.g., functions only available in Go) where that information is just in text in the description, not in the structured data, so I think just including version in the description text here is fine.